### ISSUE TITLE
add seo data

### ISSUE THREAD

**Issue by ryanschaefer41** [vigetlabs/EcoCart#15]

**Comment by doylejack74** [comment]
## Landing Page: ecocart.app

### Header

- **Page title**: EcoCart | Calculate your Environmental FOODprint
- **Meta description**: EcoCart is an educational shopping experience that gives you a personalized grade based on the environmental impact of your most common grocery items.

### Twitter Card Markup

- `twitter:card" content="summary_large_image": {use the EcoCart logo}`
- `twitter:title" content="Page Title": EcoCart | Calculate your Environmental FOODprint`
- `twitter:description" content=: EcoCart is an educational shopping experience that gives you a personalized grade based on the environmental impact of your most common grocery items.`
- `twitter:image:src" content=: {URL to EcoCart logo}`
NOT NEEDED: `<meta name="twitter:creator" content="@author_handle">`

### Facebook Card Markup

`<meta property="og:title" content="EcoCart | Calculate your Environmental FOODprint" />`
`<meta property="og:url" content="{URL to page}" />`
`<meta property="og:image" content="{URL to logo}" />`
`<meta property="og:description" content="EcoCart is an educational shopping experience that gives you a personalized grade based on the environmental impact of your most common grocery items." />`
`<meta property="og:site_name" content="EcoCart | Calculate your Environmental FOODprint" />`

## Groceries Page: ecocart.app/#groceries

### Header

- **Page title**: Groceries | EcoCart
- **Meta description**: How eco-friendly is the beef you eat? Bring your grocery list to life with an in-depth look at the environmental impact of each food item on your list!

### Twitter Card Markup

- `twitter:card" content="summary_large_image": {use the EcoCart logo}`
- `twitter:title" content="Page Title": Groceries | EcoCart`
- `twitter:description" content=: How eco-friendly is the beef you eat? Bring your grocery list to life with an in-depth look at the environmental impact of each food item on your list!`
- `twitter:image:src" content=: {URL to EcoCart logo}`
NOT NEEDED: `<meta name="twitter:creator" content="@author_handle">`

### Facebook Card Markup

`<meta property="og:title" content="Groceries | EcoCart" />`
`<meta property="og:url" content="{URL to page}" />`
`<meta property="og:image" content="{URL to logo}" />`
`<meta property="og:description" content="How eco-friendly is the beef you eat? Bring your grocery list to life with an in-depth look at the environmental impact of each food item on your list!" />`
`<meta property="og:site_name" content="Groceries | EcoCart" />`

## Receipt Page: ecocart.app/receipt

### Header

- **Page title**: View Your Impact | EcoCart
- **Meta description**: View the environmental breakdown of your chosen grocery items in your EcoCart and receive a personalized grade!

### Twitter Card Markup

- `twitter:card" content="summary_large_image": {use the EcoCart logo}`
- `twitter:title" content="Page Title": View Your Impact | EcoCart`
- `twitter:description" content=: View the environmental breakdown of your chosen grocery items in your EcoCart and receive a personalized grade!`
- `twitter:image:src" content=: {URL to EcoCart logo}`
NOT NEEDED: `<meta name="twitter:creator" content="@author_handle">`

### Facebook Card Markup

`<meta property="og:title" content="View Your Impact | EcoCart" />`
`<meta property="og:url" content="{URL to page}" />`
`<meta property="og:image" content="{URL to logo}" />`
`<meta property="og:description" content="View the environmental breakdown of your chosen grocery items in your EcoCart and receive a personalized grade!" />`
`<meta property="og:site_name" content="View Your Impact | EcoCart" />`

**Comment by nellbhealy** [comment]
> How eco-friendly is the beef you eat?

Is the word "beef" there on purpose? I see it repeated a lot but I'm pretty confused by it

**Comment by doylejack74** [comment]
> > How eco-friendly is the beef you eat?
> 
> Is the word "beef" there on purpose? I see it repeated a lot but I'm pretty confused by it

Yes! It's the description for the groceries page and I included that question as an attention-grabber to see when you search the page. The word "beef" can be replaced with any item on our list, if there's a better item to put there it can definitely be changed.

**Comment by willyd332** [comment]
Hey, so this is a single page application made with react, this means that "ecocart.app/#groceries" and "ecocart.app/receipt" don't really exists.
In other words, we can only have one Header, Facebook Card, and Twitter Card for the entire application. Do you want me to just use the top one? Or would you like to reorganize it so it has the entire site in mind?

**Comment by doylejack74** [comment]
> Hey, so this is a single page application made with react, this means that "ecocart.app/#groceries" and "ecocart.app/receipt" don't really exists.
> In other words, we can only have one Header, Facebook Card, and Twitter Card for the entire application. Do you want me to just use the top one? Or would you like to reorganize it so it has the entire site in mind?

After looking at it, I think it would work best to just use the top option for the whole site, so you can ignore the descriptions for the Groceries page and Receipt page.

**Comment by doylejack74** [comment]
## LinkedIn Card Markup (OGP)

`<meta property="og:title" content="EcoCart | Calculate your Environmental FOODprint" />`
`<meta property="og:url" content="{URL to page}" />`
`<meta property="og:image" content="{URL to logo}" />`
`<meta property="og:description" content="How eco-friendly is the beef you eat? Bring your grocery list to life with an in-depth look at the environmental impact of each food item on your list! EcoCart is an educational shopping experience that gives you a personalized grade based on the environmental impact of your most common grocery items." />`
`<meta property="og:site_name" content="EcoCart | Calculate your Environmental FOODprint" />`

**Comment by nellbhealy** [comment]
> ## LinkedIn Card Markup (OGP)
> `<meta property="og:title" content="EcoCart | Calculate your Environmental FOODprint" />`
> `<meta property="og:url" content="{URL to page}" />`
> `<meta property="og:image" content="{URL to logo}" />`
> `<meta property="og:description" content="How eco-friendly is the beef you eat? Bring your grocery list to life with an in-depth look at the environmental impact of each food item on your list! EcoCart is an educational shopping experience that gives you a personalized grade based on the environmental impact of your most common grocery items." />`
> `<meta property="og:site_name" content="EcoCart | Calculate your Environmental FOODprint" />`

We already have this code in the project 🤔

**Comment by willyd332** [comment]
Yes, this was just reopened because we have to do something different (or add something) for LinkedIn... which I haven't figured out yet

**Comment by nellbhealy** [comment]
@ryanschaefer41 This issue has been fixed and can be closed. There was an issue with the LinkedIn cache on this project, and once that was reset then LinkedIn showed the correct information
